Source/WebCore/css/parser/CSSParserContext.cpp

```diff
@@ -157,7 +158,8 @@ void add(Hasher& hasher, const CSSParserContext& context)
         | context.cssMediaProgressFunctionEnabled           << 25
         | context.cssContainerProgressFunctionEnabled       << 26
         | context.cssRandomFunctionEnabled                  << 27
-        | (uint32_t)context.mode                            << 28; // This is multiple bits, so keep it last.
+        | context.cssHasSlottedEnabled                      << 28
+        | (uint32_t)context.mode                            << 29; // This is multiple bits, so keep it last.
```

Comment:
Is 2 bits enough? Maybe the comment could specify the number of bits? Or we could have a static assertion that confirms the number of bits.

Reply:
We have 5 values in this enum so 2 bits ins't enough:
```cpp
enum CSSParserMode : uint8_t {
    HTMLStandardMode,
    HTMLQuirksMode,
    // SVG attributes are parsed in quirks mode but rules differ slightly.
    SVGAttributeMode,
    // User agent stylesheets are parsed in standards mode but also allows internal properties and values.
    UASheetMode,
    // WebVTT places limitations on external resources.
    WebVTTMode
};
```

Reply:
We should add an assert somewhere to validate that every value of CSSParserMode can be stored & loaded properly.